chart/templates/developer-hub/subscription.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-{{ if (index .Values "developer-hub") }}
+{{ if and (index .Values "developer-hub") (index .Values "developer-hub" "enabled") }}
```

Comment:
Question: Is this the best way to write the conditional? It seems hard to follow at first glance but if that is how Helm suggests doing it then it's okay

Reply:
That's interesting, cause according to [helm documentation](https://docs.helm.sh/docs/chart_template_guide/control_structures/#ifelse) it should return `false` for empty or null values. Not sure what happens if the parent value doesn't exist.

Reply:
@Jdubrick @thepetk I've simplified the conditions to just checking for the child https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/d1ad0e77d9f9b9fbe95bbc1aa0b1a56d47f78fed, going to fully verify this a bit later. Still works when enabled.

Reply:
> @Jdubrick @thepetk I've simplified the conditions to just checking for the child [d1ad0e7](https://github.com/redhat-ai-dev/ai-rhdh-installer/commit/d1ad0e77d9f9b9fbe95bbc1aa0b1a56d47f78fed), going to fully verify this a bit later. Still works when enabled.

@Jdubrick @thepetk Had to backtrack this as it errors if parent is not set, so in the first condition I changed from `index` to `hasKey` to only check if the parent key exists rather than trying to get the value: https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/b54296d7fafed04b44cb1357418bf671094c6bc6

As per what @thepetk said I think returning null on non-existence (treated as `false`) on the second statement makes sense, this is the simplest alternative method I could find with helm conditions like this.